litmus-portal/cluster-agents/subscriber/pkg/k8s/operations.go

```diff
@@ -158,52 +160,42 @@ func IsClusterConfirmed() (bool, string, error) {
 
 // ClusterRegister function creates litmus-portal config map in the litmus namespace
 func ClusterRegister(clusterData map[string]string) (bool, error) {
```

Comment:
We need to patch two things 
1. IS_CLUSTER_CONFIRMED
2. ACCESS_KEY

so, the func params can be be the access_key only and is_cluster_confirmed is a boolean(no need to add it in params)

```suggestion
func ClusterRegister(accessKey string) (bool, error) {
```